src/pydrex/core.py

```diff
@@ -364,22 +377,17 @@ def _get_strain_energy_enstatite(
     - `nucleation_efficiency` (float) — parameter controlling grain nucleation
 
     """
-    weight_factor = slip_rate_softest / crss[slip_indices[-1]] ** stress_exponent
+    # weight_factor = slip_rate_softest / crss[slip_indices[-1]] ** stress_exponent
     dislocation_density = (1 / crss[slip_indices[-1]]) ** (
         deformation_exponent - stress_exponent
-    ) * np.abs(weight_factor) ** (stress_exponent / deformation_exponent)
+    ) * np.abs(slip_rate_softest) ** (stress_exponent / deformation_exponent)
```

Comment:
What is happening here?

Reply:
Ah, I had moved the division by that factor up to an earlier line in preparation for unifying the strain energy functions (since that is the only difference between the one for enstatite and olivine). But forgot to actually merge the functions. That's done in the new commit and this calculation is moved to line 413.